### PR TITLE
Clean up Colony Home

### LINKED ISSUE #36: Clean up Colony Home
A lot of commented out code got left behind my original colony wiring.

Please take time and clean it up, refactor it properly. This includes component, styles, and everything else that's needed.

Please inspect the component before starting it, as there might be features not available yet, like #29 and decide when / how to handle that

### PR DESCRIPTION
**Pull request by jakubcolony**
## Description

This PR attempts to clean up the `ColonyHome` and related components as much as currently possible.

**Changes** 🏗

* Ported over getting the filtered domain id from search params
* Renamed the aside components to avoid confusion with identically named pages

Resolves #36

### CODE REVIEW

**Comment by jakubcolony** [comment]
I created additional issues which are related to this ticket but currently blocked by something else: #57, #58, #59 and #60.

**Review by rdig — COMMENTED** [review]
Am I doing something wrong?

When I try to run this, webpack is missing to styles files

Screenshot from 2022-11-02 12-26-43

Were they moved? Or not commited ?

**Comment by jakubcolony** [comment]
>Am I doing something wrong?

Nope, that's on me. Should be fixed now

File: src/components/common/NewActionButton/NewActionButton.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const MSG = defineMessages({
 //   };
 // }
 
-const ColonyHomeActions = (/* { colony, ethDomainId}: Props */) => {
+const NewActionButton = ({ colony, ethDomainId }: Props) => {
```
Comment (rdig) [inline src/components/common/NewActionButton/NewActionButton.tsx:62]: colony / ethdomain are not being used, please either comment them out or make sure they are used

**Review by rdig — CHANGES_REQUESTED** [review]
Mostly what I expected, but there's still some ts errors / leftover imports that need to be addressed

File: src/components/common/ColonyHome/ColonyFundingWidget/TokenItem.tsx [hunk 2]
```diff
@@ -10,13 +10,13 @@ import { Token } from '~types';
...
 interface Props {
-  currentDomainId: number;
+  // currentDomainId: number;
   token: Token;
   isTokenNative?: boolean;
   isNativeTokenLocked?: boolean;
 }
 
-const displayName = 'dashboard.ColonyHome.ColonyFundingWidget.TokenItem';
+const displayName = 'common.ColonyHome.ColonyFundingWidget.TokenItem';
```
Comment (rdig) [inline src/components/common/ColonyHome/ColonyFundingWidget/TokenItem.tsx:19]: This is not something you did here, it's just something I ran into so I'm leaving it for future reference.

We have two files named `TokenItem`. This one, and another one in `shared/TokenEditDialog/TokenItem`

We should rename one of them to keep things less confusing

**Review by rdig — APPROVED** [review]
Looks serviceable for now, but we did end up with a lot of commented out code.

I guess we'll fix it when get to each individual component.

**Review by rdig — APPROVED** [review]
All good from my end! :tada:

**Review by Julianahlk — APPROVED** [review]
LGMT 👍

**Review by danbr — APPROVED** [review]
LGTM well done.

**Review by chinins — APPROVED** [review]
Very good - I like the renaming part :)

**Pull request merged**